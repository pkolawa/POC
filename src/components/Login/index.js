import React from "react";
import Typography from "@material-ui/core/Typography";

class Login extends React.Component {
  render() {
    return (
      <div>
        <Typography align="center" variant="h2" color="secondary">
          Login Page
        </Typography>
      </div>
    );
  }
}

export { Login as default }