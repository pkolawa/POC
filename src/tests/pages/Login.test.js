import React from "react";
import { shallow } from "enzyme";
import Login from "../../pages/Login";

test("Should Render Configuration Page", () => {
  const wrapper = shallow(<Login/>);
  expect(wrapper).toMatchSnapshot();
});