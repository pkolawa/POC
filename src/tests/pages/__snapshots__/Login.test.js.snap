// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Configuration Page 1`] = `
<div>
  <WithStyles(ForwardRef(Typography))
    align="center"
    color="secondary"
    variant="h2"
  >
    Login Page
  </WithStyles(ForwardRef(Typography))>
</div>
`;
